src/api/java/io/github/opencubicchunks/cubicchunks/api/worldgen/populator/WorldGenEntitySpawner.java

```diff
@@ -57,14 +60,21 @@ public static void initialWorldGenSpawn(WorldServer world, Biome biome, int bloc
             Biome.SpawnListEntry currEntry = WeightedRandom.getRandomItem(world.rand, spawnList);
             int groupCount = MathHelper.getInt(random, currEntry.minGroupCount, currEntry.maxGroupCount);
             IEntityLivingData data = null;
-            int randX = blockX + random.nextInt(sizeX);
-            int randZ = blockZ + random.nextInt(sizeZ);
-
-            final int initRandX = randX;
-            final int initRandZ = randZ;
-
+            int randX = blockX + random.nextInt(sizeX) - 1;
```

Comment:
Why -1?

Reply:
I'm adding that 1 later so even if `random.nextInt(5)` return 0 position will be shifted anyway.